### PR TITLE
[SWP-1932] Update current ReviewsIo integration

### PR DESCRIPTION
**Pull request by mstuessy**
- Make ReviewsIo MerchantActions findReview function respect the store value from the config.
- Remove now unused class variable.

### CODE REVIEW

File: src/Providers/ReviewsIo/Request/Merchant/FindReviewRequest.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ class FindReviewRequest extends BaseRequestWrapper
      * @return Request
      */
     public function getRequest(): Request
-    {
+   {
```
Comment (adteulade) [inline src/Providers/ReviewsIo/Request/Merchant/FindReviewRequest.php]: revert changes here

**Comment by adteulade** [comment]
Just one space to add

**Comment by mstuessy** [comment]
@adteulade added the space thanks for catching that! :bow:

**Review by adteulade — APPROVED** [review]
LGTM!

**Pull request merged**